tst/com/amazon/kinesis/streaming/agent/processing/processors/DataConverterTest.java

```diff
@@ -195,12 +195,15 @@ public void testSysLogDataConverter() throws Exception {
         final String expectedLogWithoutPID = "{\"timestamp\":\"Mar 12 12:01:02\",\"hostname\":\"server4\",\"program\":\"snort\",\"processid\":null,\"message\":\"Ports to decode telnet on: 21 23 25 119\"}\n";
         verifyDataConversion(converter, logWithoutPID.getBytes(), expectedLogWithoutPID.getBytes()); 
     }
-    
-    private void verifyDataConversion(IDataConverter converter, byte[] dataBin, byte[] expectedBin) throws Exception {
-        ByteBuffer data = ByteBuffer.wrap(dataBin);
-        ByteBuffer res = converter.convert(data);
-        byte[] resBin = new byte[res.remaining()];
-        res.get(resBin);
-        Assert.assertEquals(resBin, expectedBin);
```

Comment:
Disregard the changes to the test. I was using a different API for testing. The `Assert.assertEquals()` in `testng` does compare the elements of an array. I had originally copied the test suite over and used `junit` which resulted in the discrepancy. I'll revert the change.

For future changes, what steps can be taken to run the test suite?